configuration/jsr223.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: documentation
+title: JSR223 Scripting
+---
+
+{% include base.html %}
+
+# JSR223 Scripting
+> Note: This feature is for advanced users who have or are willing to learn some programming skills and are comfortable working with the command line prompt of the operating system hosting openHAB.
```

Comment:
Honestly, I'd remove the "advanced" word here to avoid putting people off initially. If they're willing to learn, they'll get there eventually.